src/app/app.component.html

```diff
@@ -1,7 +1,11 @@
-<alm-app-header></alm-app-header>
-
-<main class="app">
-  <router-outlet></router-outlet>
+<header class="padding-bottom-10" >
```

Comment:
I think it makes more sense to have the header component contain the header tag. Same for 'main' and 'footer'. Though it may be worth looking at the Angular style guide to confirm.